### ISSUE TITLE
Connection Time Out

### ISSUE THREAD

**Issue by PemmasaniRajesh** [TooTallNate/Java-WebSocket#1186]
I'd like to set the connection timeout value for websocket client in Android. 

If I understood correctly there's a function called setConnectionLostTimeOut. I'm not sure whether we've to pass the value in milli seconds or seconds. 

I've used this function to set the timeout value. I tried setting 0 and 5 secs, but it's not working. 

It's always taking 60 secs. 

Is there any function to set the value ?

**Comment by PhilipRoman** [comment]
Yes, connection timeout is given in seconds. Setting it to zero will disable it (see https://www.javadoc.io/static/org.java-websocket/Java-WebSocket/1.5.2/org/java_websocket/AbstractWebSocket.html#setConnectionLostTimeout-int-)

I just tested it and connection timeout works correctly for me. Can you post a small program that reproduces the issue?
